### PR TITLE
doc: two minor fixes

### PR DESCRIPTION
**Pull request by drunkard**
- doc: indent style fixes
- doc: referred content should be ruleset ID

### CODE REVIEW

File: doc/rados/operations/pools.rst [hunk 1]
```diff
@@ -188,7 +188,7 @@ consider removing them when you no longer need your pool::
 
 	ceph osd pool get {pool-name} crush_ruleset
 
-If the ruleset was "123", for example, you can check the other pools like so::
+If the ruleset ID was "123", for example, you can check the other pools like so::
```
Comment (tchaikov) [inline doc/rados/operations/pools.rst]: i think we are trying to use "ruleset" to note the "id", do you find somewhere else we are using "ruleset id" in the doc, and it's causing the confusion?

Comment (drunkard) [inline doc/rados/operations/pools.rst]: Only exists in release-notes, but it confused me in my first reading, shouldn't we improve that?

Comment (drunkard) [inline doc/rados/operations/pools.rst]: Actually, we don't have enough documents on "ceph osd crush rule ..." operating topic, these commands take ruleset name as argument, so I think it's necessary to distinguish "name" and "id".

Comment (tchaikov) [inline doc/rados/operations/pools.rst]: i think we are using "rulename" to reference the name of ruleset, and use "ruleset" to reference the id of it. see http://docs.ceph.com/docs/master/rados/operations/pools/#set-pool-values and http://docs.ceph.com/docs/master/rados/operations/crush-map/#crush-map-rules. if you happen to find ruleset id anywhere else in the doc, could you point me to it? thanks.

Comment (drunkard) [inline doc/rados/operations/pools.rst]: Got it, only found in release-notes by now, get those using: `grep -Ri 'ruleset id' doc/`

**Pull request merged**